interfaces/lego-boost-sse/tools/mission/index.html

```diff
@@ -0,0 +1,284 @@
+<!doctype html>
```

Comment:
Is this file doing something here? I wasn't aware that there could be a tools directory inside a hardware interface

Reply:
We have an API now in hardwareInterfaces.js called "setTool" that allows a hardware interface to instantiate a tool on an object if it's stored in this location. I think Valentin added this feature.

Reply:
It is meant to have custom tools for specific hardware interfaces